### PR TITLE
Resource Tags - tags are missing for all resources

### PR DESCRIPTION
**Pull request by ianuragsingh**
Our TFC workspace is failing because of mandatory tags

### CODE REVIEW

**Review by niroz89 — COMMENTED** [review]
@ianuragsingh there is  no need to use the tags feature here, please use terraform `default_tags` to tag all the resources.

cc: @lawliet89

File: modules/self_managed_nodes/variables.tf [hunk 1]
```diff
@@ -88,3 +82,9 @@ variable "force_irsa" {
   type        = bool
   default     = true
 }
+
+variable "tags" {
+  description = "Tags for all resources"
+  type        = map(string)
+  default     = {}
+}
```
Comment (niroz89) [inline modules/self_managed_nodes/variables.tf:90]: Add newline at the end of file.

**Review by niroz89 — COMMENTED** [review]
@ianuragsingh please generate terraform docs.

`terraform-docs markdown --lockfile=false --recursive table --output-file README.md --output-mode inject . `

File: modules/self_managed_nodes/variables.tf (hunk 1 above)
Comment (ianuragsingh) [reply modules/self_managed_nodes/variables.tf]: @niroz89 I ran terraform fmt command but anyway still added the space as you suggested.

**Comment by niroz89** [comment]
@ianuragsingh thank you, merged.

**Pull request merged**